erizo_controller/erizoClient/src/Stream.js

```diff
@@ -280,25 +280,41 @@ Erizo.Stream = function (spec) {
         }
     };
 
+    that.muteAudio = function (audioActive, callback) {
```

Comment:
I'd say `muteAudio()/muteVideo()` is easier to understand and to develop than `mute(audio,video)` because it avoids developers to keep state of both in the same call. But I don't have a strong opinion though.